base-lite/ub/ub.go

```diff
@@ -0,0 +1,313 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"regexp"
+	"sort"
+	"strings"
+	"text/template"
+
+	"golang.org/x/sys/unix"
+)
+
+func ensure(envVar string) bool {
+	_, found := os.LookupEnv(envVar)
+	return found
+}
+
+func path(filePath string, operation string) bool {
```

Comment:
nit: Generally I would suggest you always return an `error` from a method if it can potentially encounter one and let the caller decide if they want to throw away/ignore the results. As a contrived example you can still return `(bool, error)` and the caller can ignore the error unless it's in verbose/debug mode in which case it can log it to help debug